### ISSUE TITLE
Commit #1295d93 does not seem to work for the case it was made for

### ISSUE THREAD

**Issue by Olf0** [sailfishos-patches/patchmanager#42]
Thank you @coderus for [commit #1295d93](https://github.com/sailfishos-patches/patchmanager/commit/1295d931cc5f715a38795971a629f5f04dbb955f) and releasing Patchmanager 3.0.61, including it.

To my surprise this does not make any difference (compared to Patchmanager 3.0.59) WRT the [behaviour of the RPM Patch **Shutdown screen**](https://openrepos.net/comment/28948#comment-28948) (with its v0.2-1 installed [from OpenRepos](https://openrepos.net/content/schturman/patch-shutdown-screen)):
> \[It\] causes *Settings -> Display* to emit "Could not load page" when entering this settings page (i.e., the one which is patched).
> Unapplying **Shutdown screen** lets the *Display* settings page work again, re-applying breaks it again, reproducibly ~~(that doesn't even need a *Restart services* in Patchmanager 3)~~.

Looking more closely, the Patch **Shutdown screen v0.2-1** is actually [applying fine (log)](https://github.com/sailfishos-patches/patchmanager/files/3827634/pm3.0.61-log_shutdown-screen.txt) and working (it emits the startup and shutdown messages configured before), despite breaking the *Settings -> Display* page.

Is there a chance to let [commit #1295d93](https://github.com/sailfishos-patches/patchmanager/commit/1295d931cc5f715a38795971a629f5f04dbb955f) handle this?
Would it be helpful for you to be provided with other logs or information (if so, please name them)? 

Tested on an Xperia X with SailfishOS 3.0.3 (before and after restarting Lipstick and again after a reboot) and reported for SailfishOS 3.0.x, 3.1.0 and 3.2.0 with Patchmanager 3.0.57 - .61.

**Comment by CODeRUS** [comment]
check with new build

**Comment by Olf0** [comment]
Still no difference, behaviour and PM-log is exactly as before.

`journalctl -al _COMM=jolla-settings + _EXE=/usr/sbin/patchmanager
+ _EXE=/usr/bin/patchmanager-dialog 2>&1 | tee pm3.0.61-journal_shutdown-screen.txt` (as root) outputs: 
[pm3.0.61-journal_shutdown-screen.txt](https://github.com/sailfishos-patches/patchmanager/files/3827758/pm3.0.61-journal_shutdown-screen.txt)

**Comment by Olf0** [comment]
BTW, is there a reason for not using an absolute path  `_COMM=/usr/bin/jolla-settings` (instead of `_COMM=jolla-settings`, utilising the search path) in Patchmanager 3 and when extracting from the Systemd journal?

**Comment by CODeRUS** [comment]
Thanks for testing, please check new build

**Comment by CODeRUS** [comment]
If you interested in parsing journal you can use journalctl -o json-pretty to see available fields with values

**Comment by Olf0** [comment]
> Thanks for testing, please check new build

I have troubles installing v3.0.61-1.135.1: Storeman was hanging (right after updating some other apps successfully), "updating Patchmanager forever".
Looking at the command line, any **pkcon** sub-command was hanging, too.  Apparently the **zypper** back-end was hanging.
Will reboot and retry then.

**Comment by CODeRUS** [comment]
3.0.62 fixed all problems?

**Comment by Olf0** [comment]
Slooow down, you are talking to an old man. ;)

And I have more in store. ;\

**Comment by Olf0** [comment]
Well, PM 3.0.62-1.136.1 altered the behaviour WRT this issue slightly:
1. Unchanged: **Shutdown screen** is applying fine, which seems correct judging from the log.
2. Unchanged: After that, *Restarting preloaded services* works fine.
3. Unchanged: When entering *Settings -> Display*, "Could not load page" is displayed ...
4. Different:  ... briefly, followed by Lipstick restarting hard and then all Patches are unapplied, Patchmanager's engine unloaded (Load engine entry in Patchmanager's pulley menu) and Patchmanager is in a special failure mode (Resolve failure entry in pulley).

TL;DR: Now additionally crashing Lipstick and ultimately unapplying all Patches. ;\

**Comment by CODeRUS** [comment]
yep this is default behaviour now. restarting homescreen and removing all patches if cant load any qml in lipstick or jolla-settings which are major applications of sfos :)

**Comment by Olf0** [comment]
Well, while this technically makes sense (avoiding more severe mishaps, as not starting **jolla-settings** at all, thus preventing to resolve the situation at the GUI), this is a bit "bastard operator from hell"-style. ;))

Although I believe it now sends an overly strong indication that something went wrong (solely "Could not load page" was sufficient for that, IMO), any other approach would take additional coding efforts and this solution does the job, hence closing as fixed.

**Comment by Olf0** [comment]
BTW, the other issues I was able to reproduce with various Patchmanager 3.0.61 releases (e.g. with *Lockscreen Analog Clock*) are gone with v3.0.62, hence I do not [have more in store](https://github.com/sailfishos-patches/patchmanager/issues/42#issuecomment-552221630) anymore.

Thank you!

**Comment by CODeRUS** [comment]
thank you
